rust/automerge-wasm/index.d.ts

```diff
@@ -103,6 +103,12 @@ export type PutPatch = {
   conflict: boolean
 }
 
+export type MarkPatch = {
+  action: 'mark'
+  path: Prop[],
+  marks: Mark[]
+}
+
```

Comment:
The matching `unmark` patch seems to be missing:

```suggestion

export type UnmarkPatch = {
  action: 'unmark'
  path: Prop[],
  start: number
  end: number
  key: string
}
```

Reply:
Wonderful news - I've added the unmark patch.  Good catch.